app/javascript/react/components/ParkShowContainer.js

```diff
@@ -19,15 +56,40 @@ const ParksShowContainer = props => {
       })
       .then(response => response.json())
       .then(response => {
-        setParkInfo(response.park)
+        if (response.review) {
+          setReviews([...reviews, response.review])
+        } else {
+          setErrors(response.errors)
+        }
+      })
+      .catch(error => console.error(`Error in fetch: ${error.message}`));
+    }
+
+    useEffect(() => {
+      fetch(`/api/v1/parks/${parkId}/reviews`)
+      .then(response => {
+        if (response.ok) {
+          return response
+        } else {
+          let errorMessage = `${response.status} (${response.statusText})`,
+          error = new Error(errorMessage)
+          throw error
+        }
+      })
+      .then(response => response.json())
+      .then(reviewData => {
+        setReviews(reviewData.reviews)
       })
       .catch(error => console.error(`Error in fetch: ${error.message}`));
     }, [])
 
```

Comment:
there shouldnt be two fetch requests for info here. this one should be merged with the first one. I may merge this in just so you can move forward, but then would ask you to refactor this fetch request into a combined fetch in the future.

Reply:
pending